tuf_conformance/repository_simulator.py

```diff
@@ -151,7 +158,7 @@ def rotate_keys(self, role: str) -> None:
         self.root.roles[role].keyids.clear()
         self.signers[role].clear()
         for _ in range(0, self.root.roles[role].threshold):
-            signer = CryptoSigner.generate_ecdsa()
+            signer = self._rsa_signers.pop()
```

Comment:
maybe wrap `_rsa_signers.pop()` in a `checkout_key()` method, so it becomes easier to parameterize key types? & also to error in a helpful way if more than N keys are needed